libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -236,9 +239,21 @@ - (UIView *)createTitleItem {
     return item;
 }
 
+- (BOOL)shouldAutorotate {
+    return YES;
+}
+
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
+    return UIInterfaceOrientationMaskAll;
```

Comment:
Why did we change this? I'd prefer if we locked down to portrait mode as the default within the SDK.